3people/7week/p12978.py

```diff
@@ -0,0 +1,21 @@
+from collections import deque
+import sys
+
+def solution(N, road, K):
+  nodes = {}
+  dist = {i:sys.maxsize for i in range(1, N+1)}
+  dist[1] = 0
+
+  for v1, v2, d in road:
+    nodes[v1] = nodes.get(v1, []) + [(v2, d)]
+    nodes[v2] = nodes.get(v2, []) + [(v1, d)]
```

Comment:
```suggestion
from collection import defaultdict
nodes = defaultdict(list)
for v1, v2, d in road:
    nodes[v1].append((v2, d))
    nodes[v2].append((v1, d))
```
defaultdict를 쓰면 더 간편하게 작성할 수 있습니다.